backend/search/views.py

```diff
@@ -70,9 +69,19 @@ def search_view(request):
         query.components.add(*components_obj)
         query.initialize()
 
+    if new_query:
+        try:
+            run_search_query.delay(query.pk)
+        except run_search_query.OperationalError:
+            # No connection with message broker - run synchronously
+            run_search_query.apply((query.pk,))
```

Comment:
It's nice to have a fallback here, but I wonder how we will be alerted in case celery goes down

Reply:
Good point. My idea here is that a user who wants to run GrETEL on their own machine can do so without using Celery. I will make an issue about what to do if services go down.